### PR TITLE
Ask for App Store review after 5 successful conversions

### PR DESCRIPTION
**Pull request by sindresorhus**
And only ask once! I think this is a good balance of not being intrusive.

### CODE REVIEW

File: Gifski/Constants.swift [hunk 1]
```diff
@@ -6,4 +6,5 @@ extension NSColor {
 
 extension Defaults.Keys {
 	static let outputQuality = Defaults.Key<Double>("outputQuality", default: 1)
+	static let totalConversionsCount = Defaults.Key<Int>("totalConversionsCount", default: 0)
```
Comment (boyvanamstel) [inline Gifski/Constants.swift]: I prefer `successfulConversionsCount` or just `conversionsCount`. I think the key itself implies that it’s the total amount.

**Comment by sindresorhus** [comment]
This worked out well. Already got 2 new reviews:

Screen Shot 2019-04-06 at 05 50 24
Screen Shot 2019-04-06 at 05 50 32

**Comment by boyvanamstel** [comment]
That’s cool. Nice work. :)

**Comment by sindresorhus** [comment]
The reviews keeps pouring in. This was definitely worth doing.

Screen Shot 2019-04-10 at 12 19 25

Nice work, team! 🙌

**Pull request merged**